.evergreen/scripts/generate_config.py

```diff
@@ -337,8 +345,49 @@ def create_load_balancer_variants():
     return variants
 
 
+def create_compression_variants():
+    # Compression tests - standalone versions of each server, across python versions, with and without c extensions.
+    # PyPy interpreters are always tested without extensions.
+    host = "rhel8"
+    task_names = dict(snappy=[".standalone"], zlib=[".standalone"], zstd=[".standalone !.4.0"])
+    variants = []
+    for ind, (compressor, c_ext) in enumerate(product(["snappy", "zlib", "zstd"], C_EXTS)):
+        expansions = dict(COMPRESSORS=compressor)
+        handle_c_ext(c_ext, expansions)
+        base_name = f"{compressor} compression"
+        python = CPYTHONS[ind % len(CPYTHONS)]
+        display_name = get_display_name(base_name, host, python=python, **expansions)
+        variant = create_variant(
+            task_names[compressor],
+            display_name,
+            python=python,
+            host=host,
+            expansions=expansions,
+        )
+        variants.append(variant)
+
+    other_pythons = PYPYS + CPYTHONS[ind:]
+    for compressor, python in zip_cycle(["snappy", "zlib", "zstd"], other_pythons):
+        expansions = dict(COMPRESSORS=compressor)
+        handle_c_ext(c_ext, expansions)
+        base_name = f"{compressor} compression"
+        display_name = get_display_name(base_name, host, python=python, **expansions)
+        variant = create_variant(
+            task_names[compressor],
+            display_name,
+            python=python,
+            host=host,
+            expansions=expansions,
+        )
+        variants.append(variant)
+
+    return variants
+
+
 ##################
 # Generate Config
 ##################
 
-generate_yaml(variants=create_load_balancer_variants())
+variants = create_compression_variants()
+# print(len(variants))
```

Comment:
```suggestion
```

Reply:
Should we keep it in this final merge into master?

Reply:
I'll just have to keep adding it back in every time while I'm building out these variants, doesn't seem worth doing

Reply:
Can you let me know when the last shrub conversion PR is then? That way I know when any debugging code should be gone.

Reply:
When I close the main ticket lol, a long way off